regression/test_http_tables.py

```diff
@@ -0,0 +1,156 @@
+from helpers import tf_cfg, deproxy, chains
+from framework import tester
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class HttpTablesTest(tester.TempestaTest):
```

Comment:
First, why the test is in `regression` group? This group of tests was initially created to group all bitty tests of complicated and very specific issues in one place. This new tests extends existing `sched/test_http` test, so should be placed somewhere near it. Since we have no `http` scheduler any more, i suggest to move both tests into a new directory `http_rules` to make them easy to find by the directive name.

Secondly, please provide description, which tests cases are covered by the test. It's important since it looks like the test actually repeats some cases of the existing `sched.test_http` test.

Finally, I don't see any test case for `block` action.

Reply:
Corrected.